### DISCUSSION TITLE
Does this project support LiteLLM ? If no then why not

### DISCUSSION THREAD

**Discussion by Greatz08** [stitionai/devika#303] [General]
LiteLLM is pretty great project which helps us in using 100+ LLM easily with same structure in which we just need to feed base api url and model name and api key and start local proxy server which easily help us in using many so many llms easily. In description of project ollama is mentioned which is fine but those who dont wanna run locally but wanna use particular llm which has api but is not supported natively by this project cannot use llm until and unless you integrate litellm project with this one

**Comment by erkinalp** [comment]
check https://github.com/opendevin/opendevin instead, that one has litellm support
